simulator/mips/mips_instr.h

```diff
@@ -391,7 +394,7 @@ class BaseMIPSInstr
 
         Execute function = &BaseMIPSInstr::execute_unknown;
     protected:
-        BaseMIPSInstr( MIPSVersion version, uint32 bytes, Addr PC);
+        BaseMIPSInstr( MIPSVersion version, uint32 bytes, Addr PC, uint64 sequence_id);
```

Comment:
I don't like adding more parameters to the ctor.
It would be better to have a standalone setter function:
```c++
    void set_sequence_id(uint64 id) { / * ... */ }
```